src/Services/Microsoft.AspNetCore.SystemWebAdapters.Services.csproj

```diff
@@ -0,0 +1,35 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFrameworks>net6.0;net472</TargetFrameworks>
+    <IsPackable>true</IsPackable>
+    <EnablePackageValidation>true</EnablePackageValidation>
+    <RootNamespace>Microsoft.AspNetCore.SystemWebAdapters</RootNamespace>
+  </PropertyGroup>
+
+  <PropertyGroup Label="Package Information">
+    <Description>A collection of services to provide support for enabling configuration for Microsoft.AspNetCore.SystemWebAdapters.</Description>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <InternalsVisibleTo Include="Microsoft.AspNetCore.SystemWebAdapters.FrameworkServices"/>
+    <InternalsVisibleTo Include="Microsoft.AspNetCore.SystemWebAdapters.CoreServices"/>
```

Comment:
This tells me things aren't factored very cleanly. What's internal that the other projects need, and why doesn't it live in those assemblies instead?

Reply:
I've moved any implementation shared into a shared source pattern. Now, the only thing in the abstractions are the actual contracts